src/err/mod.rs

```diff
@@ -559,6 +559,13 @@ impl<'a> std::fmt::Display for PyDowncastError<'a> {
     }
 }
 
+#[cfg(feature = "anyhow")]
+impl From<anyhow::Error> for PyErr {
+    fn from(err: anyhow::Error) -> Self {
+        crate::exceptions::PyRuntimeError::new_err(format!("{:?}", err))
+    }
+}
```

Comment:
From a code organisation perspective I'd quite like this to go into a new file `src/conversions/anyhow.rs`, and it would be nice to also add tests for it!